TweetMate/src/main/java/com/ybsystem/tweetmate/utils/ExceptionUtils.java

```diff
@@ -0,0 +1,115 @@
+package com.ybsystem.tweetmate.utils;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import twitter4j.TwitterException;
+
+/**
+ * Utils class for creating error message
+ */
+public class ExceptionUtils {
+
+    private static final Map<Integer, String> ERROR_MESSAGE_MAP;
+
+    private ExceptionUtils() {
+    }
+
+    /**
+     * Return error message that matches the error code
+     *
+     * @param t Throwable instance
+     * @return Error message
+     */
+    public static String getErrorMessage(Throwable t) {
+
+        if (!(t instanceof TwitterException)) {
+            return "読み込みエラー";
+        }
+
+        TwitterException e = (TwitterException) t;
+
+        if (e.isCausedByNetworkIssue()) {
+            return "ネットワーク接続エラー";
+        }
+
+        if (e.exceededRateLimitation() || e.getErrorCode() == 88) {
+            return "API制限に到達しました。\n"
+                    + e.getRateLimitStatus().getSecondsUntilReset() / 60 + "分"
+                    + e.getRateLimitStatus().getSecondsUntilReset() % 60 + "秒後に解除されます。";
+        }
+
+        String errorMessage = ERROR_MESSAGE_MAP.get(e.getErrorCode());
+        if (errorMessage == null) {
+            return "読み込みエラー";
+        } else {
+            return errorMessage;
+        }
+    }
+
+    static {
+        Map<Integer, String> map = new HashMap<>();
+        map.put(3, "無効なパラメータです。");
+        map.put(13, "場所を検索できません。");
+        map.put(17, "プロファイルが見つかりません。");
+        map.put(32, "認証できませんでした。");
+        map.put(34, "対象が存在しません。");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "対象が存在しません。" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)